src/Microsoft.TestPlatform.CommunicationUtilities/LengthPrefixCommunicationChannel.cs

```diff
@@ -63,6 +63,13 @@ public Task Send(string data)
         /// <inheritdoc />
         public Task NotifyDataAvailable()
         {
+            // TODO: Review the comment below, because it says something different than what is
```

Comment:
Is it a todo for you before the PR gets merged or a later change that needs to be done? If the 2nd, could we create a ticket and link it to the todo?

Reply:
Yes, please do it 😏